data/json/items/gun/40mm.json

```diff
@@ -16,12 +41,62 @@
     "range": 30,
     "dispersion": 180,
     "sight_dispersion": 120,
-    "aim_speed" : 6,
+    "aim_speed": 6,
     "recoil": 270,
     "durability": 8,
     "burst": 10,
     "reload": 400,
     "magazines": [ [ "40mm", [ "belt40mm" ] ] ],
+    "valid_mod_locations": [[ "accessories", 4 ],[ "grip", 1 ],[ "mechanism", 4 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]],
     "flags": "MOUNTED_GUN"
+  },
+  {
+    "id": "rm802",
+    "copy-from": "launcher_base",
+    "type": "GUN",
+    "name": "RM802 grenade launcher",
+    "description": "The Rivtech RM802 pump-action 40mm grenade launcher was designed to be a step ahead of single-shot grenade launchers by allowing sustained heavy firepower.  However, its capacity is fairly limited so it must be used with careful planning.",
+    "weight": 3120,
+    "volume": 6,
+    "price": 2920000, "//": "Either Rivtech or milspec would make it Expensive.  Combined, you could buy a half-decent car for the price.",
+    "bashing": 8,
+    "to_hit": -1,
+    "material": [ "superalloy", "ceramic" ],
+    "color": "dark_gray",
```

Comment:
Indeed, but some uniformity would actually be useful. It makes sense elsewhere (eg .`brown` for wooden items) but is rather arbitrary here.